recipe/meta.yaml

```diff
@@ -31,12 +26,6 @@ requirements:
     - cmake
```

Comment:
Seems like Python is still in the build reqs.

Reply:
Yep, just re-added as per earlier comment.